### PR TITLE
remove faraday gemspec version lock

### PR DESCRIPTION
**Pull request by grantspeelman**
* change the faraday dependency lock version  
* add rspec and a simple test of sms
* increase to v2.1.0

### CODE REVIEW

**Comment by msxavi** [comment]
👍🏼

File: message_media_messages.gemspec [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
   s.email = 'developers@messagemedia.com'
   s.homepage = 'https://developers.messagemedia.com'
   s.license = 'Apache-2.0'
   s.add_dependency('logging', '~> 2.0')
-  s.add_dependency('faraday', '~> 0.10')
-  s.add_dependency('test-unit', '~> 3.1.5')
+  s.add_dependency('faraday', '>= 0.10.0')
   s.add_dependency('certifi', '~> 2016.9', '>= 2016.09.26')
-  s.add_dependency('faraday-http-cache', '~> 1.2', '>= 1.2.2')
+  s.add_dependency('faraday-http-cache', '>= 1.2.0')
```
Comment (s01ipsist) [inline message_media_messages.gemspec:13]: shall we retain the `1.2.2` minimum dependency previously specified?
```suggestion
  s.add_dependency('faraday-http-cache', '>= 1.2.2')

```

**Review by s01ipsist — COMMENTED** [review]
What is the command used to execute the specs and can this be documented on readme?
`bundle exec rspec` ?

**Review by s01ipsist — COMMENTED** [review]
👍 lgtm

**Pull request closed without merging**